packages/shared/src/domain/patient/patient-import/create.ts

```diff
@@ -30,8 +30,8 @@ export function createPatientImport({
   };
 
   const {
-    rerunPdOnNewDemographics = false,
-    triggerConsolidated = false,
+    rerunPdOnNewDemographics = true,
+    triggerConsolidated = true,
```

Comment:
Important change, now all bulk imports will rerun PD on new demo and trigger consolidated automatically